nolabs/modules/small_molecules_design/start_learning.py

```diff
@@ -0,0 +1,16 @@
+from reinvent_microservice import Configuration, ApiClient, DefaultApi
+
+from nolabs.infrastructure.settings import Settings
+
+
+class StartLearningExperimentFeature:
+    def __init__(self, settings: Settings):
+        self._settings = settings
+
+    async def handle(self, experiment_id: str):
+        configuration = Configuration(
+            host=self._settings.reinvent_host,
+        )
+        with ApiClient(configuration=configuration) as client:
+            api_instance = DefaultApi(client)
+            api_instance.learning_jobs_job_id_learning_post(job_id=experiment_id)
```

Comment:
Having jpb id as an experiment id will cause us some trouble later when we merge all of the labs and tools into one single lab. I'd advise to change it now to have a separate id for a job (but can do later as refactoring). 

Arguments in favor of having a separate job id instead of experiment id:

1) Running a sampling job != running a learning job (but here we have the same id for both of them. I know that there's a job name, but still it's not great imo)

2) In the future when we merge models in one lab from which we do all the research, if we are still passing experiment id as a job id here will cause us troubles when looking at the dashboard of running jobs (which we will obviously have, given the scale of research and computations required to produce good results)

3) Job management (stop a job/continue a job/change the job order in the queue) will be awkward as well with this design approach

Reply:
1) Sampling is a continuation of the learning job. You use the same model checkpoints, same target folder, same protein, just switching the 'mode' of the job.

2-3) We will change this in a future. We don't know our feature set and architectural approach in advance, so I just picked the simplest approach to satisfy requirements of the current task.